src/components/Footer/index.tsx

```diff
@@ -61,7 +61,7 @@ export default function Footer() {
   ];
 
   return (
-    <footer className="flex justify-center w-full z-5 backdrop-blur px-4 pt-2 bg-transparent shadow-2xl">
+    <footer className="flex justify-center w-full z-5 px-4 pt-2 bg-transparent shadow-2xl">
```

Comment:
The `backdrop-blur` attribute is here to indicate to the user they are at the end of the page